src/ui/base-page-storyEdit.js

```diff
@@ -149,6 +142,51 @@ https://raw.githubusercontent.com/fluid-project/sjrk-story-telling/master/LICENS
             clearAutosave: {
                 funcName: "sjrk.storyTelling.base.page.storyEdit.clearAutosave",
                 args: ["{that}.options.pageSetup.storyAutosaveKey"]
+            },
+            initializeStory: {
```

Comment:
Please add code comments to explain how this and following functions are used at different stages of the auto save and publish.